src/Build/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
```

Comment:
please keep this comment. It is added by APICompat/PackageValidation v7 and upwards.